tests/test_readers_writers/test_read_carroucell.py

```diff
@@ -0,0 +1,31 @@
+# This is a sample Python script.
+
```

Comment:
Just remove this line above and its perfect!

Reply:
done !